px-map-behavior-marker-group.es6.js

```diff
@@ -672,25 +672,26 @@
       this._bindAndOpenPopup(evt.layer);
     },
 
-    _handleSingleMarkerDoubleTap(evt) {
+    _handleSingleMarkerDoubleClick(evt) {
       const latLng = L.latLng(evt.layer.getLatLng());
       const {lat, lng} = latLng;
       const detail = {
         latLng: latLng,
         lat: lat,
         lng: lng
       };
-      this.fire('px-map-marker-group-double-tapped', detail);
+      this.fire('px-map-marker-group-double-clicked', detail);
```

Comment:
Last thought on this.... lets make it more specific and do `px-map-marker-group-marker-double-clicked` to distinguish from a cluster being clicked, which could be implemented in the future. Weird long name, but it is specific.